site/docs/concepts/advanced/logs-stats.md

```diff
@@ -1,5 +1,46 @@
 # Logs and statistics
 
+Flow collects logs and statistics of catalog tasks to aid in debugging and refinement of your workflows.
+
 :::caution
-Page in progress. Check back soon.
-:::
\ No newline at end of file
+Access to statistics is still a work in progress. For now, this documentation deals strictly with logs.
+:::
+
+## Logs
+
+Each organization, or tenant, that uses Flow has a `logs` collection under the global `ops` prefix.
+For the tenant Acme Co, it would have the name `ops/acmeCo/logs`.
+
+These can be thought of as standard application logs:
+they store information about events that occur at catalog build time.
```

Comment:
"build time" isn't accurate here. We don't publish build logs in this way, but only runtime logs. Maybe a better way to say this would be: they store events that occur while running your shards.